src/wrappers/themis/jsthemis/index.d.ts

```diff
@@ -0,0 +1,60 @@
+export declare class KeyPair {
+    public(): Uint8Array;
+    private(): Uint8Array;
+}
+
+export declare class SymmetricKey {
+    constructor();
+}
+
+export declare class SecureCellSeal {
+    static withKey(key: SymmetricKey): SecureCellSeal;
+    static withPassphrase(passphrase: string): SecureCellSeal;
+    encrypt(plaintext: Uint8Array, context?: Uint8Array): Uint8Array;
+    decrypt(encrypted: Uint8Array, context?: Uint8Array): Uint8Array;
+}
+
+export declare interface SecureCellTokenProtectResult {
+    data: Uint8Array;
+    token: Uint8Array;
+}
+
+export declare class SecureCellTokenProtect {
+    static withKey(key: SymmetricKey): SecureCellTokenProtect;
+    encrypt(plaintext: Uint8Array, context?: Uint8Array): SecureCellTokenProtectResult;
+    decrypt(encrypted: Uint8Array, token: Uint8Array, context?: Uint8Array): Uint8Array;
+}
+
+export declare class SecureCellContextImprint {
+    static withKey(key: SymmetricKey): SecureCellContextImprint;
+    encrypt(plaintext: Uint8Array, context: Uint8Array): Uint8Array;
+    decrypt(encrypted: Uint8Array, context: Uint8Array): Uint8Array;
+}
+
+export declare class SecureMessage {
+    constructor(privateKey: Uint8Array | null, publicKey: Uint8Array | null);
+    sign(message: Uint8Array): Uint8Array;
+    verify(message: Uint8Array): Uint8Array;
+    encrypt(message: Uint8Array): Uint8Array;
+    decrypt(message: Uint8Array): Uint8Array;
+}
+
+type GetPublicKeyCallback = (peerID: Uint8Array) => Uint8Array | null;
+
+export declare class SecureSession {
+    constructor(peerID: Uint8Array, privateKey: Uint8Array, getPublicKeyCallback: GetPublicKeyCallback);
+    isEstablished(): boolean;
+    connectRequest(): Uint8Array;
+    sendToPeer(message: Uint8Array): void;
```

Comment:
I didn't find these functions in secureSession.cpp to find result type. Where this function is declared?

Reply:
Removed. Thank you. I did not write tests for SecureSession.